air_quality_cli/constants.py

```diff
@@ -63,7 +65,7 @@
         ),
     ),
     (
-        range(301, 501),
+        range(301, 1001),
```

Comment:
@nsn39  Can you share the resource, where the range from 300-1000 is considered hazardous? For most the cap was 500.

**Reference:**  [Indices by location](https://en.wikipedia.org/wiki/Air_quality_index#Indices_by_location)

Reply:
![aqi_value1](https://user-images.githubusercontent.com/34716481/109426672-f3051d80-7a16-11eb-8d35-89959c207f50.png)
 This is the place where the aqi exceeds 500.